operations/helm/charts/mimir-distributed/templates/query-frontend/autoscaling.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.query_frontend.kedaAutoscaling.enabled }}
```

Comment:
There is some convention to name the files as `<name>-<resource_kind>.yaml`. Can you name these e.g. `query-frontend-scaledObject.yaml` or `query-frontend-so.yaml`?